### PR TITLE
mp-b2c-product-offer-availabilities-endpoint-tests

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description

mp-b2c-product-offer-availabilities-endpoint-tests
https://spryker.atlassian.net/browse/CC-16358

### CODE REVIEW

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    I send a GET request:    /concrete-products/${merchants.spryker.concrete_product_with_offer_sku}/product-offers
+    AND Save value to a variable:    [data][0][id]    offerId
+    When I send a GET request:    /product-offers/${offerId}/product-offer-availabilities
+    Then Response status code should be:    200
+    And Response reason should be:    OK
+    And Each array element of array in response should contain property with value:    [data]    type    product-offer-availabilities
+    And Each array element of array in response should contain property:    [data]    id
+    And Each array element of array in response should contain property:    [data]    attributes
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    isNeverOutOfStock
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    availability
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot:22]: please check that isNeverOutOfStockm and availability are boolean

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    And Each array element of array in response should contain nested property:    [data]    [attributes]    quantity
```
Comment (OKrivtsova) [inline tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot:23]: please check quantity is numeric

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot:22]: DONE

File: tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot (hunk 2 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/mp_b2c/glue/product_offer_endpoints/product_offer_availabilities/positive.robot:23]: DONE
getting quantity value datatype in string. So I validated quantity datatype str.

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**